UI/ChatGPT/ChatGPT/ChatGPT/Services/ChatService.cs

```diff
@@ -0,0 +1,89 @@
+using ChatGPT.Business;
+using OpenAI.Managers;
+using OpenAI;
+using System.Runtime.CompilerServices;
+using OpenAI.ObjectModels.RequestModels;
+using OpenAI.ObjectModels;
+using System.Text;
+using OpenAI.ObjectModels.ResponseModels;
+using OpenAI.Interfaces;
+
+namespace ChatGPT.Services;
+public class ChatService(IChatCompletionService client) : IChatService
+{
+	private readonly IChatCompletionService _client = client;
+
+	public async ValueTask<ChatResponse> AskAsync(string prompt, CancellationToken ct = default)
+	{
+		try
+		{
+			var request = new ChatCompletionCreateRequest()
+			{
+				Messages = new List<ChatMessage> { ChatMessage.FromUser(prompt) },
+				Model = Models.Gpt_3_5_Turbo
```

Comment:
We should include a system prompt and previous messages in the conversation to get a more contextual result. Because right now this application will be like talking to a goldfish...

We should have a system prompt (usually the first one) instructing the model what it is and what to do with the messages and include assistant/user messages. Like in this sample: https://github.com/betalgo/openai?tab=readme-ov-file#chat-gpt-sample

Reply:
It does make sense. It can now have the context of the whole conversation... Added in next commit.

![image](https://github.com/unoplatform/Uno.Samples/assets/114886335/98d1d893-f02e-4d2b-8f0e-114a39a02a5c)

@dr1rrb can you check if changes in last commit makes sense?